web/projects/ui/src/app/pages/apps-routes/app-interfaces/app-interfaces.page.ts

```diff
@@ -126,3 +100,65 @@ export class AppInterfacesItemComponent {
     await toast.present()
   }
 }
+
+function getAddresses(
+  serviceInterface: ServiceInterfaceWithHostInfo,
+): MappedAddress[] {
+  const host = serviceInterface.hostInfo
+  const addressInfo = serviceInterface.addressInfo
+  const username = addressInfo.username ? addressInfo.username + '@' : ''
+  const suffix = addressInfo.suffix ? '/' + addressInfo.suffix : ''
```

Comment:
I think the suffix might already include a /

Reply:
Why is that. Are you expecting package devs to write it, or is the BE adding it universally to the string?

Reply:
It gets built by the sdk from other components, and that function adds a leading / if appropriate